File: Assets/GetBody.cs
using UnityEngine;
using System.Collections;

public class GetBody : MonoBehaviour {

	private Sprite body;

	void Start () {
		this.gameObject.GetComponent<SpriteRenderer>().sprite = GameObject.Find("Enemy" + Random.Range(1,4)).gameObject.GetComponent<SpriteRenderer>().sprite;
	}
	
	void Update () {
	
	}
}

File: Assets/Script/PlayScreen/GetBody.js
#pragma strict

static var type : int;

function Start () {
	type = Random.Range(0,Score.point/10) + 1;
	if(type > 4){
		type = 4;
	}
	this.gameObject.GetComponent(SpriteRenderer).sprite = GameObject.Find("Enemy" + Random.Range(1,5)).gameObject.GetComponent(SpriteRenderer).sprite;
}

function Update () {

}